engine/src/flutter/shell/platform/embedder/BUILD.gn

```diff
@@ -497,6 +497,8 @@ if (is_mac && !embedder_for_target) {
       rebase_path(sources[0]),
       "--destination",
       rebase_path(outputs[0]),
+      "--buildmode",
+      flutter_runtime_mode,
```

Comment:
I'm not actually sure how/where the Embedder framework is used, but I added to it too so it's consistent.